client/src/pages/Review/ReviewsList/ReviewsListParent.tsx

```diff
@@ -1,15 +1,60 @@
-import React from "react";
-import { ReviewsSeed } from "./ReviewsSeed";
-import ReviewsList from "./ReviewsList";
-import { ReviewsListParent } from "./ReviewsListStyles";
+import React, { ReactNode, useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { ReviewList, ReviewsWrapper } from "./ReviewsListStyles";
+import { getRestaurant } from "../../../utils/serverCalls";
+
+interface Reviews {
+  name: string;
+  children: ReactNode;
+}
+
+interface Restaurant {
+  author_name: string;
+  language: string;
+  rating: number;
+  text: string;
+  time: number;
+  reviews: Reviews[];
+}
 
 function ReviewListParent() {
+  const { place_id } = useParams();
+
+  const [restaurantData, setRestaurantData] = useState<Restaurant[]>([]);
+
+  useEffect(() => {
+    async function fetchData() {
+      try {
+        const data = await getRestaurant(place_id);
```

Comment:
We can also use the `getReviewsByRestaurant()` api call here, to get our proprietary reviews, rather than just the google reviews.